images/content/kit/battery_charger_hke4_diagram.svg

```diff
@@ -1,644 +1 @@
s" font-size="28" text-anchor="middle" writing-mode="lr" x="524.40942" y="1003.7795">Lead</tspan></text><path d="m471.25984 853.93688 53.1496 53.14961v28.34645" fill="none" marker-start="url(#a)" stroke="#000" stroke-width="2.834646"/><text font-family="Bitstream Vera Sans" font-size="28" letter-spacing="0" text-anchor="middle" word-spacing="0" x="620.07874" y="655.51172"><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" x="620.07874" y="655.51172">Charging</tspan><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" x="620.07874" y="690.51172">Lead</tspan></text><path d="m595.27559 754.72428 24.80315-24.80315v-28.34645" fill="none" marker-start="url(#a)" stroke="#000" stroke-width="2.834646"/><text style="font-size:28;letter-spacing:0;word-spacing:0;text-anchor:middle;font-family:Bitstream Vera Sans;writing-mode:lr" x="301.18109" y="1034.6455"><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="301.18109" y="1034.6455">Start/Stop Button</tspan></text><path d="m301.1811 960.2361v46.063" fill="none" marker-start="url(#a)" stroke="#000" stroke-width="2.834646"/><text style="font-size:28;letter-spacing:0;word-spacing:0;text-anchor:middle;font-family:Bitstream Vera Sans;writing-mode:lr" x="88.58268" y="899.99994"><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="88.58268" y="899.99994">Charge</tspan><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="88.58268" y="939.99994">Current</tspan><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="88.58268" y="979.99994">Knob</tspan></text><path d="m265.74803 892.91326-35.43307 35.43307h-74.40945" fill="none" marker-start="url(#a)" stroke="#000" stroke-width="2.834646"/><path d="m237.40157 797.24396-35.43307-35.43307h-46.06299" fill="none" marker-start="url(#a)" stroke="#000" stroke-width="2.834646"/><text style="font-size:28;letter-spacing:0;word-spacing:0;text-anchor:middle;font-family:Bitstream Vera Sans;writing-mode:lr" x="99.212601" y="733.46448"><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="99.212601" y="733.46448">Charge</tspan><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="99.212601" y="773.46448">Status</tspan><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="99.212601" y="813.46448">LEDs</tspan></text><text style="font-size:28;letter-spacing:0;word-spacing:0;text-anchor:middle;font-family:Bitstream Vera Sans;writing-mode:lr" x="432.28345" y="574.01569"><tspan font-family="Bitstream Vera Sans" font-size="28" text-anchor="middle" writing-mode="lr" x="432.28345" y="574.01569">Power Lead</tspan></text><path d="m265.74803 563.3857h70.86614" fill="none" marker-start="url(#a)" stroke="#000" stroke-width="2.834646"/></g></svg>
```

Comment:
The SVGs here are the source files for their diagrams. Suggest that we not "optimise" these source images.
SVGs will be served GZIPped (or better), so they'll end up about the same size as the optimised versions anyway meaning that there's likely no substantial gain here.